juju/juju
# Change subnet.AvailabilityZone in state from a string to []string and rename.

## Description of change

Change subnet.AvailabilityZone in state from a string to []string and rename.
Update tests accordingly. Added an upgrade state. Migration piece in the
works.

## QA steps

Bootstrap a cloud with availability zones.  Run juju upgrade.  Check the subnets collection with juju-db.bash to verify AvailabilityZone is now AvailabilityZones and a slice.

Do the same with model migration.
